### DISCUSSION TITLE
Using Xpore with Nanopore latest RNA chemistry -RNA004

### DISCUSSION THREAD

**Discussion by Annie-GW** [GoekeLab/xpore#224] [Q&A]
I would like to know whether xpore can be used to identify m6A modification site for direct RNA sequencing data generated using the new chemistry RNA004. If not, is there a plan to update xpore for this ? Thank you!

**Comment by shayanjl** [comment]
I do not see any clear xPore documentation stating that RNA004 is currently supported. The current xPore docs still describe the general differential modification workflow, but I could not find an RNA004-specific update there. ([[GitHub](https://github.com/GoekeLab/xpore?utm_source=chatgpt.com)][1])

So the cautious answer is that RNA004 support is not clearly documented in xPore at the moment. If you need an RNA004-ready workflow for m6A specifically, the GoekeLab tool **m6Anet** does document an RNA004 model in its release notes. ([[GitHub](https://github.com/GoekeLab/m6anet?utm_source=chatgpt.com)][2])

It would be helpful if the maintainers could confirm whether xPore has been validated for RNA004 or whether support is planned.

[1]: https://github.com/GoekeLab/xpore?utm_source=chatgpt.com "GitHub - GoekeLab/xpore: Identification of differential RNA modifications from nanopore direct RNA sequencing"
[2]: https://github.com/GoekeLab/m6anet?utm_source=chatgpt.com "GitHub - GoekeLab/m6anet: Detection of m6A from direct RNA-Seq data"
